packages/apps/helpdesk/src/components/molecules/TicketCard/TicketCard.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+import {
+  Badge,
+  Card,
+  Icon,
+  LabelText,
+  Text,
+  useThemeColor,
+} from '@axelor/aos-mobile-ui';
+import {useTranslator, formatDateTime} from '@axelor/aos-mobile-core';
+import Ticket from '../../../types/ticket';
+
+interface TicketCardProps {
+  style?: any;
+  ticketSeq: string;
+  subject: string;
+  progressSelect: string;
+  duration: number;
+  deadlineDateT: string;
+  responsibleUser: string;
+  assignedToUser: string;
+  prioritySelect: number;
+  statusSelect: number;
+  allTicketType?: any;
+  ticketType?: any;
+  onPress: () => void;
+}
+const TicketCard = ({
+  style,
+  ticketSeq,
+  subject,
+  progressSelect,
+  duration,
+  deadlineDateT,
+  responsibleUser,
+  assignedToUser,
+  prioritySelect,
+  statusSelect,
+  allTicketType,
+  ticketType,
+  onPress,
+}: TicketCardProps) => {
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+
+  const colorType = useMemo(() => {
+    const colorIndex = allTicketType?.findIndex(
+      status => status.id === ticketType?.id,
+    );
+    return getStyles(Ticket.getTypeColor(colorIndex, Colors))?.colorBadge;
+  }, [Colors, allTicketType, ticketType?.id]);
+
+  const colorStatus = useMemo(() => {
+    return getStyles(Ticket.getStatusColor(statusSelect, Colors))?.colorBadge;
+  }, [Colors, statusSelect]);
+
+  const borderStyle = useMemo(() => {
+    return getStyles(Ticket.getPriorityColor(prioritySelect, Colors))?.border;
+  }, [prioritySelect, Colors]);
+
+  return (
+    <TouchableOpacity onPress={onPress} activeOpacity={0.9}>
+      <Card style={[styles.container, borderStyle, style]}>
+        <View style={styles.leftContainer}>
+          <Text style={styles.txtImportant}>
+            {I18n.t('Helpdesk_Progress')}n°{ticketSeq}
```

Comment:
This is not the right translation here, I think Helpdesk_Ticket is more revelant and please add space between translation and "n°"